spec/requests/v1/pacbio/libraries_spec.rb

```diff
@@ -586,6 +592,8 @@
 
       it 'publishes a message' do
         expect(Messages).to receive(:publish).with(library.sequencing_runs, having_attributes(pipeline: 'pacbio'))
+        expect(Emq::Publisher).to receive(:publish).with(library.primary_aliquot,
```

Comment:
I think I raised this one with Seena's pull request but if the publish message is tested fully elsewhere then we only need to test that publish has been called.